### DISCUSSION TITLE
[Agenda]2023/08/04

### DISCUSSION THREAD

**Discussion by bjwswang** [kubebb/core#200] [Community Meetings]
Please list the items you want to discuss in this meeting with this template!

## Template

```markdown
## [number] Title
/kind propose

### Description

What is this item for?
```

For example:

```markdown
## 1. propose a template for Pull Request
/kind propose

### Description

Here I want to propse a template when making  PR in kubebb projects. 
```

**Comment by bjwswang** [comment]
## 1. Kickoff this community meeting
/kind propose

### Description

From now on, we start our community meeting even though only you and me joined. 

1. Meeting at Every Friday 4:00pm-5:00pm. https://github.com/kubebb/core/issues/180
2. Before you join in this meeting, file in what you want to discuss in [`Discussion`](https://github.com/kubebb/core/discussions/categories/community-meetings)(If you can provide more details that will be much better)
3. Follow our Code of Conduct https://github.com/kubebb/core/blob/main/CODE_OF_CONDECT.md

## 2. Review the website document structure

/kind docs

### Description

I just restructured the document but need further review.

**Comment by bjwswang** [comment]
## 1. Request a Showcase of PR/Git common usage
/kind help

### Description

Show some common ways/rules to make a PR.

**Reply by bjwswang** [reply]
@Abirdcfly

**Reply by Abirdcfly** [reply]
https://gist.github.com/Abirdcfly/b942d4e9d44f3fb71344b92b979c8d40

**Comment by bobsongplus** [comment]
Menus CRD  design need discuss in detail.
------
/kind propose

### Description

 deprecated `parentOwnerReferences ` will introduce an incompatible problem. https://github.com/kubebb/core/issues/91

**Reply by bjwswang** [reply]
@bobsongplus Can you share more details about this problem?

**Comment by bjwswang** [comment]
## 1. Share more details about Rating design

/kind share

### Description

I want to share more details about `Rating` including:
-  Overall workflow
- How to cooperate with Lowcode platform
- Next move
   -  1. Define Rating CRD
   -  2. Develop Rating controller: watch on PipelineRun

**Reply by bjwswang** [reply]
## Workflow
![rating_flow drawio](https://github.com/kubebb/core/assets/30621793/1b1f902a-cdd2-477a-800f-38aa80f6c90a)


## Subtasks

1. https://github.com/kubebb/core/issues/209
2. https://github.com/kubebb/core/issues/210
